frb_codegen/src/generator/dart/mod.rs

```diff
@@ -407,7 +407,11 @@ fn generate_api_func(func: &IrFunc, ir_file: &IrFile) -> GeneratedApiFunc {
     };
     let parse_sucess_data = if (f.is_static_method()
         && f.struct_name().unwrap() == {
-            if let IrType::StructRef(IrTypeStructRef { name, freezed: _ }) = &func.output {
+            if let IrType::StructRef(IrTypeStructRef {
```

Comment:
"Submodule allo-isolate added at a24225" - seems that we need to remove it

(this comment is unrelated to this line of code, just cannot find elsewhere to comment)